### PR TITLE
Remove deprecated extensions (Optimizely and Google Analytics)

### PR DESCRIPTION
**Pull request by v-mohithgc**
The source code of these extensions (Optimizely and Google Analytics) will be removed as it's no longer maintained/ supported and is marked as deprecated.

### CODE REVIEW

File: DEPRECATION.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+Creation: 2023-02-24
```
Comment (ozanovus) [inline DEPRECATION.md:3]: Which date format you're preferring?  Here it is `YYYY-MM-DD` but in the table and latest update it is DD-MM-YYYY or MM-YYYY. 

Please pick one for all document for consistency.

Comment (v-mohithgc) [reply DEPRECATION.md]: sorry, my bad.
will be going with DD-MM-YYYY format.

**Review by ozanovus — APPROVED** [review]

**Review by max-zaytsev — APPROVED** [review]

**Review by DmitriiBobreshev — APPROVED** [review]

**Review by v-nagarajku — APPROVED** [review]
LGTM

**Pull request merged**